Makefile

```diff
@@ -0,0 +1,15 @@
+init: migrate
```

Comment:
can change this command to build instead?

Reply:
Sure.

Reply:
```suggestion
build: migrate.up
```